src/components/Nav.css

```
@@ -1,6 +1,6 @@
 .nav {
   display: flex;
-  padding: 2.0625rem 12.5rem 2rem 12.5rem;
+  padding: 2rem 12.5rem 2rem 12.5rem;
```

단축속성 활용하시면 다음과 같이 작성할 수 있습니다.
```suggestion
  padding: 2rem 12.5rem;
```